community/presentations.md

```diff
@@ -8,16 +8,22 @@ redirect_from: /docs/presentations
 
 Conference and Meetup Presentations on Spinnaker
 
+### <a href="http://2018.opendevconf.com/" target="_blank">OpenDev CI/CD</a>
+**Videos from OpenDev Conference, Vancouver, BC** \| May 22-23, 2018
+
+* [How Dev/Ops Evolves as the Cloud Does](https://www.youtube.com/watch?v=LjjfWBrvyps){:target="\_blank"}
```

Comment:
I think this is "DevOps" (no `/`)? Also, curious about`target="_blank"`? The videos further down the page don't open in a new window, as far as I can make out?

/cc @skim1420

Reply:
Yeah, omit the slash. ![slash](https://i.imgur.com/EF69hLd.jpg)

Reply:
@demobox , I've been adding the target="_blank", per #743 
It's not non-controversial ;), and I'm open to argument. 
Mean time, I'll add those on this page to be consistent.